instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py

```diff
@@ -96,6 +124,8 @@ def process_request(self, request):
         if self._excluded_urls.url_disabled(request.build_absolute_uri("?")):
             return
 
+        request.start_time = time.time()
```

Comment:
I think we should try to reduce chances of collision with user code and other 3rd party libraries by prefixing something like `_otel_` to these attributes. So they would become `_otel_start_time` and `_otel_labels`.